Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.9.5
+FROM python:3.9.9-buster

```

Comment:
why buster?

Reply:
To keep the same Debian version. Current default changed to bullseye which breaks something :)